tools/dialignr/dialignr.R

```diff
@@ -0,0 +1,13 @@
+library(DIAlignR)
+
+args <- commandArgs(trailingOnly = FALSE)
+
+data_path <- paste0(args[7], "/data/")
+runs_file_path <- paste0(args[7], "/runs.txt")
```

Comment:
or just feed the path strings directly to the function .. there is no need to define extra variables.

Reply:
The data folder is job_working_directory and the R script is in the tool directory. I've tried different things like symlink the R script in the job_working directory, but that didn't work either..

Reply:
> The data folder is in the job_working_directory and the R script is in the tool directory. I've tried different things like symlink the R script in the job_working directory, but that didn't work either..

True. But the suggested change should work, because the the current working dir (`pwd`) will be the job working dir.

Reply:
@npinter you could pass the `pwd` into the R script.

Reply:
> 
> 
> @npinter you could pass the `pwd` into the R script.

That's what I did!

> True. But the suggested change should work, because the the current working dir (`pwd`) will be the job working dir.

That was to easy... changed it 👍 Thanks!